File: baekjoon/4504/Main.c
#include <stdio.h>

int main()
{

    int n;
    scanf("%d", &n);

    while(1){
        int temp;
        scanf("%d", &temp);
        if(temp == 0){
            break;
        }
        
        if(temp % n == 0){
            printf("%d is a multiple of %d.\n", temp, n);
        }else{
            printf("%d is NOT a multiple of %d.\n", temp, n);
        }
    }
  
}

File: baekjoon/4504/Main.py
N = int(input())
ans = ""

while True:
    temp = int(input())
    if(temp == 0):
        break;
    
    if(temp % N == 0):
        print("%d is a multiple of %d." %(temp,N))
    else:
        print("%d is NOT a multiple of %d." %(temp,N))
    